src/components/icons/ATATSVGIcon.vue

```diff
@@ -13,12 +13,14 @@ import {stringObj } from "types/Global";
 import Vue from "vue";
 
 import { Component, Prop, PropSync } from "vue-property-decorator";
+import ArrowLeft from "@/components/icons/ArrowLeft.vue"
```

Comment:
Same icon is in PR 1017 for AT-7881 - I named it `ArrowBack` -- we should probably only keep one in the codebase